frontend/src/components/public_group/PublicGroupHero.js

```diff
@@ -7,22 +7,45 @@ export default class PublicGroupHero extends React.Component {
 
   renderHeroStatistics()
   {
-    const { group } = this.props;
-    if (!group.backers.length) return;
-    const backerCount = filterCollection(group.backers, {tier: 'backer'}).length;
-    const sponsorCount = filterCollection(group.backers, {tier: 'sponsor'}).length;
+    const { group, i18n } = this.props;
     const yearlyIncome = group.yearlyIncome / 100;
     const formattedYearlyIncome = yearlyIncome && formatCurrency(yearlyIncome, group.currency, { compact: true, precision: 0 });
-    return (
+
+    let tierCountString;
+    const tierCountStringArray = group.tiers.slice()
+    .sort((tierA, tierB) => tierB.range[0] - tierA.range[0])
+    .map(tier => {
+      const count = filterCollection(group.backers, {tier: tier.name}).length;
+      return (count) ? `${count} ${count === 1 ? tier.name : `${tier.name}s`}` : '';
+    })
+    .filter(x => x);
+
+    if (tierCountStringArray.length > 1)
+    {
+      if (tierCountStringArray.length > 2)
+      {
+        const lastCountString = tierCountStringArray.pop();
+        tierCountString = `${tierCountStringArray.join(', ')} ${i18n.getString('and')} ${lastCountString}`
```

Comment:
Note that I don't think this approach scales very well. You're lucky French is rather similar to English in grammar structure, but this won't work if word order is any different. I'd have the i18n templates be whole sentences, not individual words.